docs/architecture/hotspot-makers/mcc/hotspot-audit-process.mdx

```diff
@@ -73,20 +73,21 @@ samples and payments. **Missing information will result in significant delays.**
 
 ### LoRaWAN Capable Hotspots
 
-1. The Hotspot should arrive running as it would be at a customer's location.
-2. Provide instructions for stopping and starting Packet Forwarder.
-3. Provide the location of the Packet Forward configuration file `global_conf.json`.
-4. Provide the path to the `gateway-rs` executable (ie: `helium_gateway`)
-5. Provide the architecture/distribution used by the CPU.
+1. The Hotspot should arrive running as it would be at a customer's location. Y/N
```

Comment:
```suggestion
1. The Hotspot should arrive running as it would be at a customer's location. **Y/N**
```
This yes/no doesn't match the formatting of the other y/n questions.

If yes, should the manufacturer answer the other questions?

Reply:
No, they typically provide the path to the files asked for. "Provide the location..."
The Y/N can be removed.